qutip/qobj.py

```diff
@@ -1834,25 +1867,30 @@ def trans(self):
         return out
 
     def extract_states(self, states_inds, normalize=False):
-        """Qobj with states in state_inds only.
+        """:obj:`.Qobj` with states in `states_inds` only.
```

Comment:
broken formatting of states_inds

Reply:
Ok. I will put it back to what it is supposed to be. I was a bit confused about this because in some cases, variables / methods appear in double quotes, some appear in italics and some cases use double backticks for monospaced text.